src/v2i-hub/RSUHealthMonitorPlugin/src/RSUHealthMonitorPlugin.cpp

```diff
@@ -27,6 +27,7 @@ namespace RSUHealthMonitor
         for (auto rsuConfig : _rsuConfigListPtr->getConfigs())
         {
             auto rsuStatusJson = _rsuWorker->getRSUStatus(rsuConfig.mibVersion, rsuConfig.rsuIp, rsuConfig.snmpPort, rsuConfig.user, rsuConfig.authPassPhrase, rsuConfig.securityLevel, SEC_TO_MICRO);
+            rsuStatusJson[RSUIdentifierKey] = rsuConfig.RSUIdentifier;
```

Comment:
Why are we pulling this from the configuration, are we not setting this on the RSU?

Reply:
This is to directly sent as part of the RSU status payload.